libraries/libfc/libraries/boringssl/CMakeLists.txt

```diff
@@ -3,6 +3,13 @@ target_compile_options(fipsmodule PRIVATE -Wno-error)
 target_compile_options(crypto PRIVATE -Wno-error)
 target_compile_options(decrepit PRIVATE -Wno-error)
 
+# boringssl's forced-on debug symbols interfere with reproducibility due to path differences; disable on Release builds
+if(CMAKE_BUILD_TYPE STREQUAL "Release")
```

Comment:
Checking `CMAKE_BUILD_TYPE` like this isn't ideal at all and maybe even borderline anti-pattern. But I'm not really sure what other indicator to use of "no thanks I don't want debug symbols"